cosmo_tester/test_suites/ha/ha_cluster_scenarios_test.py

```diff
@@ -19,6 +19,7 @@
 from cosmo_tester.framework.examples.hello_world import HelloWorldExample
 from cosmo_tester.framework.cluster import CloudifyCluster
 from .ha_helper import HighAvailabilityHelper as ha_helper
+from . import community_skip
```

Comment:
Also, isn't there a way in pytest to skip a whole module? I feel like there's got to be.